goss-testing/scripts/check_clock_skew.sh

```diff
@@ -49,6 +49,11 @@ for node in $nodes; do
   pdsh_node_args="$pdsh_node_args -w $node"
 done
 
+for try in {1..3}; do
```

Comment:
should break out of the loop when no clock skew is detected?

Reply:
We could.  I was just trying to keep the changes short to address the intermittent issues seen with the test.  I don't think it'e entirely necessary despite it adding 30 seconds to the test.

Reply:
I think we should either break out on success or shorten the sleep to like 2s.  Or was 10 chosen based on real-life observations?

Reply:
Just an arbitrary value I chose.  I'll reduce to 2

Reply:
Fixed to `sleep 2`